Kiosk/App/AppDelegate.swift

```diff
@@ -53,11 +64,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             Stripe.setDefaultPublishableKey(keys.stripeProductionPublishableKey)
         }
 
-        ARAnalytics.setup(withAnalytics: [
-            ARHockeyAppBetaID: keys.hockeyBetaSecret,
-            ARHockeyAppLiveID: keys.hockeyProductionSecret,
-            ARSegmentioWriteKey: keys.segmentWriteKey
-        ])
+        ARAnalytics.setup(
+            withAnalytics: [
+                ARHockeyAppBetaID: keys.hockeyBetaSecret,
+                ARHockeyAppLiveID: keys.hockeyProductionSecret,
+                ARSegmentioWriteKey: keys.segmentWriteKey
+            ]
+        )
```

Comment:
Do you think that `withAnalytics` should be inline like it was before?

Reply:
Treat initial named params as something worth inlining makes sense to me.

Reply:
Yeah, ConditionalGroup is needed here :( At least I couldn’t find a way without. You can look at the logic around expandLastArg for reference